etc/p3-config/p3-config.yml

```diff
@@ -357,9 +358,53 @@ p3_subnet_internal:
   allocation_pool_start: "10.0.0.2"
   allocation_pool_end: "10.0.0.254"
 
+# P3 octavia network name.
+p3_network_octavia_name: "p3-octavia"
+
+# P3 octavia network.
+p3_network_octavia:
+  name: "{{ p3_network_octavia_name }}"
+  project: "service"
+  provider_network_type: "vlan"
+  provider_physical_network: "physnet1"
+  provider_segmentation_id: 148
+  shared: False
+  # Subnet configuration.
+  subnets:
+    - "{{ p3_subnet_octavia }}"
+
+# P3 octavia subnet.
+p3_subnet_octavia:
+  name: "{{ p3_network_octavia_name }}"
+  project: "service"
+  cidr: "10.48.0.0/16"
+  enable_dhcp: true
+  allocation_pool_start: "10.48.253.2"
+  allocation_pool_end: "10.48.253.254"
+
 # P3 Bulk Data Network (BDN) network name.
 p3_network_bdn_name: "p3-bdn"
 
+p3_security_groups:
+  - name: p3-octavia-sg
+    description: "Octavia security group"
+    project: service
+    rules:
+      - protocol: udp
+        port_range_min: 5555
+        port_range_max: 5555
+        remote_ip_prefix: "0.0.0.0/0"
+      - protocol: tcp
+        port_range_min: 9443
+        port_range_max: 9443
+        remote_ip_prefix: "0.0.0.0/0"
+      - protocol: tcp
+        port_range_min: 22
+        port_range_max: 22
+        remote_ip_prefix: "10.208.0.0/30"
+      - protocol: icmp
+        remote_ip_prefix: "0.0.0.0/0"
```

Comment:
Have you missed the log ports? UDP 10514 and 20514

Reply:
Added. Didnt miss it but seems to work without.